src/platform/site-wide/mega-menu/index.js

```diff
@@ -17,10 +18,64 @@ import Main from './containers/Main';
  * @param {Redux.Store} store The common store used on the site
  */
 export default function startMegaMenuWidget(data, store) {
```

Comment:
This additional logic for the entry point file is due to us removing the desktop react app when you're on mobile + vice versa when you're on desktop, right? I wonder if it would be faster + less code to manage if we left both components mounted always but then conditionally render them based on the viewport width